README.md

```diff
@@ -57,6 +57,16 @@ You can also run the discord portion using docker if you prefer, it simply needs
 Example:
 `docker run -p 8123:8123 -e DISCORD_BOT_TOKEN=<YourTokenHere> denverquane/amongusdiscord`
 
+## Deploy to Heroku
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+
+The app will fail the first time you deploy since the `DISCORD_BOT_TOKEN` is not set. To fix this:
+
+1. Create a new Config var in your Heroku app's settings with the key `DISCORD_BOT_TOKEN` and the value as your bot token obtained from the pre-installation step.
+2. Restart all dynos
+
+To connect to this deployment, create a `host.txt` file in the same folder as the  `amonguscapture.exe` file with the contents `https://<host>`, where the host is your Heroku app URL and restart `amonguscapture.exe` if its already running.
```

Comment:
probably should revise to the format `http(s)://<host>:<port>`, where port is the port provided in ENV, or 8123 by default.

Reply:
@denverquane The port is not really required in this case since Heroku exposes it on the standard 443 for https and just specifying the hostname is enough or `<hostname>:443` but you can make the call on that.